src/PowerShell/WinGetPowerShell.sln

```diff
@@ -0,0 +1,849 @@
+﻿

```

Comment:
I know you love this so maybe you can add the new powershell projects, the server and processor 😁